### PR TITLE
[GR-911] 'Add All' button

### PR DESCRIPTION
**Pull request by avarsava**
Adds all the runs to the Run ID box. Doesn't update the graphs, but still triggers the loading screen (I suppose any callback will).

### CODE REVIEW

**Comment by callunity** [comment]
Could the button be moved to be inline with or below the "Run" label? That way, it'll be clear that it applies only to this table (which will be more relevant when we've got several multiselect dropdowns ie. for runs, projects, instrument models).

Could the styling also be changed so it doesn't look like the Update and Download buttons which have  [global/more-dramatic] effects?

File: application/dash_application/views/pre_exome.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
     # Buttons
     'update-button',
     'download-button',
+    'all',
```
Comment (callunity) [inline application/dash_application/views/pre_exome.py]: Perhaps `all-runs` instead of just `all`?

Comment (avarsava) [reply application/dash_application/views/pre_exome.py]: done

**Comment by avarsava** [comment]
> Could the button be moved to be inline with or below the "Run" label? That way, it'll be clear that it applies only to this dropdown (which will be more relevant when we've got several multiselect dropdowns ie. for runs, projects, instrument models).
> 
> Could the styling also be changed so it doesn't look like the Update and Download buttons which have [global/more-dramatic] effects?

I've added a bit of CSS, please let me know what you think.

**Pull request merged**